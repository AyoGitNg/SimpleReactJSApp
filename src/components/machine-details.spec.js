import React from 'react';
import renderer from 'react-test-renderer';

import machineDetails from './machine-details';

test('Machine Details', () => {
	const machine = {
		id: '99ade105-dee1-49eb-8ac4-e4d272f89fba',
		name: 'Machine 1',
		ip_address: '127.0.0.31',
		health: 60,
		onClick: jest.fn()
	};

	const tree = renderer.create(<machineDetails health={machine.health} ip_address={machine.ip_address}
												 onClick={machine.onClick} id={machine.id}
												 name={machine.name}/>);
	expect(tree).toMatchSnapshot();

});
