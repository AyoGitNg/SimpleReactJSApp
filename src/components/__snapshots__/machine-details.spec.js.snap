// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Machine Details 1`] = `
<machineDetails
  health={60}
  id="99ade105-dee1-49eb-8ac4-e4d272f89fba"
  ip_address="127.0.0.31"
  name="Machine 1"
  onClick={[MockFunction]}
/>
`;
